<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Authorization</class>
 <widget class="QDialog" name="Authorization">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="LogInButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="registrationButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Новый аккаунт</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordAccept_label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>177</y>
     <width>145</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Подтвердить пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordAccept">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>177</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="status_label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="login_label">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>111</y>
     <width>43</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="loginEdit">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>111</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>97</x>
     <y>144</y>
     <width>53</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordEdit">
   <property name="geometry">
    <rect>
     <x>173</x>
     <y>144</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
